extensions/env-injector/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "env-exporter"
+version = "0.1.0"
+edition = "2021"
```

Comment:
```suggestion
edition = "2024"
```

Reply:
The Rust 2024 edition [is not released yet](https://doc.rust-lang.org/edition-guide/rust-2024/index.html).

Reply:
Also, editions are the way that Rust [guarantees backward compatibility](https://doc.rust-lang.org/edition-guide/editions/index.html#what-are-editions).